src/OrbitGl/FunctionsDataView.cpp

```diff
@@ -108,23 +108,8 @@ std::string FunctionsDataView::GetValue(int row, int column) {
       return absl::StrFormat("%lu", function.size());
     case kColumnModule:
       return orbit_client_data::function_utils::GetLoadedModuleName(function);
-    case kColumnAddress: {
-      const ProcessData* process = app_->GetTargetProcess();
-      // If no process is selected, that means Orbit is in a disconnected state aka displaying a
-      // capture that has been loaded from file. CaptureData then holds the process
-      if (process == nullptr) {
-        const CaptureData& capture_data = app_->GetCaptureData();
-        process = capture_data.process();
-        CHECK(!app_->IsCaptureConnected(capture_data));
-      }
-      CHECK(process != nullptr);
-      const ModuleData* module =
-          app_->GetModuleByPathAndBuildId(function.module_path(), function.module_build_id());
-      CHECK(module != nullptr);
-      return absl::StrFormat("0x%llx", orbit_client_data::function_utils::GetAbsoluteAddress(
-                                           function, *process, *module)
-                                           .value_or(0));
-    }
+    case kColumnAddress:
```

Comment:
Maybe rename the constant as well to `kColumnAddressInModule` (although these names are not very precise).